nidn/fdtd_integration/compute_spectrum_fdtd.py

```diff
@@ -153,3 +170,28 @@ def _average_along_detector(signal):
             s[2] += p[2] / len(signal[i])
         avg[i] = s
     return avg
+
+
+def _summed_thickness_times_permittivity(thicknesses, permittivity):
+    summed_permittivity = 0
+    for i in range(len(thicknesses)):
+        summed_permittivity += thicknesses[i] * permittivity[:, :, i].real
+    return summed_permittivity
+
+
+def _check_sufficient_timesteps(
+    cfg: DotMap, thicknesses, wavelength, permittivity, scaling
+):
+    recommended_timesteps = int(((
+        cfg.FDTD_free_space_distance
+        + 3 * _summed_thickness_times_permittivity(thicknesses, permittivity)
+        + 2 * wavelength * 1e6
+    ) / (1 / torch.sqrt(torch.tensor(2.0)) * (1 / scaling))).item()// 1)
```

Comment:
```suggestion
    ) / (1 / (scaling*torch.sqrt(torch.tensor(2.0))).item())
```
I think this can be simplified a bit? (brackets probably wrong hard to do here)

Reply:
Yeah can simplify